index.html

```diff
@@ -1,11 +1,115 @@
+              fill="#252424"
+            />
+            <rect x="71" y="5" width="228" height="90" rx="18" stroke="#252424" stroke-width="10" />
+            <path
+              d="M64 48.3238C66.6667 49.8634 66.6667 53.7124 64 55.252L44.1818 66.694C41.5152 68.2336 38.1818 66.3091 38.1818 63.2299V40.3459C38.1818 37.2667 41.5152 35.3422 44.1818 36.8818L64 48.3238Z"
+              fill="#252424"
+            />
+            <path
+              d="M10.6061 19.9697C10.6061 45.4243 16.4394 51.7879 42.4242 51.7879"
+              stroke="#252424"
+              stroke-width="8"
+            />
+            <circle cx="10.6061" cy="17.8485" r="10.6061" fill="#252424" />
+          </svg>
+        </h1>
+        <h2>A language for authoring state machines.</h2>
+      </section>
```

Comment:
```suggestion
      </section>
      <p>
      StateML is a language for authoring state machines and statecharts. StateML schematics can be compiled too and interpreted by any language with a supporting library.
      </p>
```

how about this? to start with